tests/unit_tests/data_steward/validation/ehr_union_test.py

```diff
@@ -78,22 +80,21 @@ def setUpClass(cls):
 
     def setUp(self):
         self.hpo_ids = [self.FAKE_SITE_1, self.FAKE_SITE_2]
+        self.mock_bq_client_patcher = mock.patch(
+            'validation.ehr_union.BigQueryClient')
+        self.mock_bq_client = self.mock_bq_client_patcher.start()
+        self.addCleanup(self.mock_bq_client_patcher.stop)
```

Comment:
I did not know about `addCleanup()` until now. How you are using it is really cool! I had written many integration tests that needed multiple patching to multiple test cases. I could definitely have made those simpler if I had known this patch start/stop + addCleanup technic.